en/contributing/build.md

```diff
@@ -223,7 +223,7 @@ The main steps are:
    ```
 1. Enter the following command in your host's terminal:
    ```sh
-   docker run -it -v $HOME/<path-to-dronecore-repo>/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore bash
+   docker run --rm -it -v $HOME/<path-to-dronecore-repo>/DroneCore:/home/docker1000/src/DroneCore:rw dronecore/dronecore bash
    ```
    > **Note** The `-v` flag maps a directory on your host (left side) to a path in
```

Comment:
How about we add a note about the --rm flag, as below:

```
> **Note** The `-v` flag maps a directory on your host (left side) to a path in the container (right side). 
> You need to specify the left-side path to the DroneCore repository on your host and the container path must be set as above. 
> The `--rm` automatically cleans up leftover docker containers after you exit the docker container.
```

Reply:
Done 9a1896c